shell/browser/ui/message_box_win.cc

```diff
@@ -172,22 +170,26 @@ int ShowTaskDialogUTF16(NativeWindow* parent,
       config.dwFlags |= TDF_USE_COMMAND_LINKS;  // custom buttons as links.
   }
 
+  int button_id;
+  bool cb_checked = false;
```

Comment:
Minor style nit: can eliminate this temporary state by doing,

`return std::make_pair(button_id, checkbox_checked ? verificationFlagChecked : false);`